### PR TITLE
Admin place

### PR DESCRIPTION
**Pull request by yunicho**

### CODE REVIEW

File: src/main/resources/static/js/admin.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+
+function listPlaces(i, places) {
+    let place_info = places["matjip_address"]
+    console.log(places)
+    let html_temp = `<tr id="place-${i}">
+                        <td>${places['id']}</td>
+                        <td>${places['name']}</td>
+                        <td>${places['addr']}</td>
+                        <td> <button class="button" onclick="deleteAdminPlace(${places['id']})"> DELETE </button> </td>
```
Comment (leverest96) [inline src/main/resources/static/js/admin.js:25]: places.id로 통일 필요해보여요!
@yunicho @devpcjin @seonmikimm @jin2424

File: src/main/java/com/cracker/place/dto/AdminPlaceListRequestDto.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.cracker.place.dto;
+
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
```
Comment (jin2424) [inline src/main/java/com/cracker/place/dto/AdminPlaceListRequestDto.java:7]: 좋아요

File: src/main/resources/static/js/admin.js (hunk 1 above)
Comment (devpcjin) [inline src/main/resources/static/js/admin.js:25]: abc

**Review by jin2424 — COMMENTED** [review]
좋아요

File: src/main/java/com/cracker/place/dto/AdminPlaceListRequestDto.java (hunk 2 above)
Comment (jin2424) [reply src/main/java/com/cracker/place/dto/AdminPlaceListRequestDto.java:7]: > 좋아요

네

**Pull request merged**